internal/driver/driver_test.go

```diff
@@ -203,6 +203,57 @@ func TestParse(t *testing.T) {
 	}
 }
 
+// TestAppendComment tests that the flag '-comment foo' will append a comment to the profile.
+func TestAppendComment(t *testing.T) {
+
+	savedVariables := pprofVariables.makeCopy()
+	defer func() { pprofVariables = savedVariables }()
+
+	tempFile, err := ioutil.TempFile("", "profile_proto")
+	if err != nil {
+		t.Fatalf("Error in TempFile: %v", err)
```

Comment:
Nit: the most used wording for test failures is `t.Fatalf("failed to do something: %v", err)` or `t.Fatalf("ioutil.TempFile(): got error %v, want no error", err)` -- consider using one of these.

Reply:
change these to "function(): got error %v"

Reply:
These are not changed yet? Let me know with a comment like "Made all changes, the PR is ready to be reviewed again" when you make all the changes. Holding off the review for now.